### PR TITLE
IR Sensor working

### PR DESCRIPTION
**Pull request by SantiQ0905**

### CODE REVIEW

File: src/main/Subsystems/Storage/Storage.h [hunk 1]
```diff
@@ -22,10 +23,6 @@ class Storage : public frc2::SubsystemBase {
 	void shuffleboardPeriodic();
 private:
 	OverTalonFX storageMotor{ 24, ControllerNeutralMode::Brake, false, "rio" };
-
-	frc::DigitalInput forwardSensor{ 0 }; //2
-	frc::DigitalInput backSensor{ 2 }; // 0
-
-	frc::Debouncer m_debouncer{ 35_ms, frc::Debouncer::DebounceType::kBoth };
-	bool noteOnForwardCache = false;
+	rev::ColorSensorV3 OverColorSensor{frc::I2C::Port::kOnboard};
```
Comment (AJahueyM) [inline src/main/Subsystems/Storage/Storage.h]: Change name to colorSensor

File: src/main/Subsystems/Storage/Storage.cpp [hunk 2]
```diff
@@ -14,19 +14,23 @@ void Storage::setVoltage(units::volt_t voltage) {
 
 bool Storage::isNoteOnForwardSensor() {
 	// return !forwardSensor.Get();
-	return noteOnForwardCache;
+	//return noteOnForwardCache;
+	return IRvalue > 20;
```
Comment (AJahueyM) [inline src/main/Subsystems/Storage/Storage.cpp]: Add the threshold for note detection to constant file

**Review by AJahueyM — CHANGES_REQUESTED** [review]

**Comment by AJahueyM** [comment]
@SantiQ0905

File: src/main/Subsystems/Storage/Storage.h (hunk 1 above)
Comment (SantiQ0905) [reply src/main/Subsystems/Storage/Storage.h]: Fixed

File: src/main/Subsystems/Storage/Storage.cpp (hunk 2 above)
Comment (SantiQ0905) [reply src/main/Subsystems/Storage/Storage.cpp]: Fixed

**Comment by SantiQ0905** [comment]
> @SantiQ0905

Fixed

**Pull request merged**